charts/flannel-2.0.0/Chart.yaml

```diff
@@ -8,3 +8,5 @@ sources:
 - https://github.com/flannel-io/flannel
 version: 2.0.0
 icon: icons/flannel-icon.svg
+annotations:
```

Comment:
I noticed that the flannel does not have "current" in it's image tag https://github.com/oracle-cne/catalog/blob/main/charts/flannel-2.0.0/values.yaml#L12.